### PR TITLE
s390x: enable s390x by use newer qemu bin in v0.78

### PR DESCRIPTION
**Pull request by huoqifeng**
Signed-off-by: Qi Feng Huo <huoqif@cn.ibm.com>

### CODE REVIEW

File: .semaphore/semaphore.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ blocks:
       - if [ -z "${SEMAPHORE_GIT_PR_NUMBER}" ]; then make push ARCH=$ARCH CONFIRM=true; fi
       matrix:
       - env_var: ARCH
-        values: ["amd64", "arm64","armv7", "ppc64le"]
+        values: ["amd64", "arm64","armv7", "ppc64le" "s390x"]
```
Comment (mgleung) [inline .semaphore/semaphore.yml:65]: we're missing a `,` between `ppc64le` and `s390x`

Comment (huoqifeng) [reply .semaphore/semaphore.yml]: Thank you @mgleung , my fault, fix it.

**Review by mgleung — APPROVED** [review]

**Pull request merged**